homeassistant/components/blink/__init__.py

```diff
@@ -150,10 +174,32 @@ async def async_unload_entry(hass, entry):
     hass.services.async_remove(DOMAIN, SERVICE_REFRESH)
     hass.services.async_remove(DOMAIN, SERVICE_SAVE_VIDEO_SCHEMA)
     hass.services.async_remove(DOMAIN, SERVICE_SEND_PIN)
+    hass.services.async_remove(DOMAIN, SERVICE_BULK_DOWNLOAD)
 
     return True
 
 
+async def async_bulk_download_service(hass, entry, call):
+    """Handle call to bulk downloader."""
+    since = call.data["since"]
+    camera_list = call.data.get(CONF_NAME, "all")
+    video_path = call.data[CONF_PATH]
+    if not hass.config.is_allowed_path(video_path):
+        _LOGGER.error("Can't write %s, no access to path!", video_path)
```

Comment:
If we raise an exception with the message as exception argument it will show in the frontend. Use a 'ValueError`.